### ISSUE TITLE
onClick on wrapper

### ISSUE THREAD

**Issue by honzahana** [BeSite/jQuery.TosRUs#65]
Hi,

looks like TosRUs contains bug when wrapper `onClick : 'close'` is set. Example: 

```
$("article a[href$='.jpg']).tosrus({
		keys : {
			prev  : true,
			next  : true,
			close : true
		},
		infinite : true,
		wrapper : {
			onClick : 'close'
		}
	});
```

Firebug produces: 
```
this.nodes.$uibg is undefined
```

Chrome:
```
Uncaught TypeError: Cannot read property 'add' of undefined
```

**Comment by FrDH** [comment]
Thank you for reporting the issue, I'll fix it in the next version.

**Comment by FrDH** [comment]
Should be fixed in version 2.4.1
